src/utils/Constants.ts

```diff
@@ -46,32 +46,74 @@ export const FORMATS = {
   rpm: 'rpm'
 };
 
+export const PURL_TYPES = {
+  alpine: 'alpine',
+  cargo: 'cargo', //cargo == crates == rust
+  chocolatey: 'chocolatey',
+  clojars: 'clojars',
+  cocoapods: 'cocoapods',
+  composer: 'composer', //packagist website but composer format, php language
+  conan: 'conan',
+  conda: 'conda',
+  cran: 'cran',
+  debian: 'deb',
+  gem: 'gem',
+  github: 'github',
+  golang: 'golang',
+  maven: 'maven',
+  npm: 'npm',
+  nuget: 'nuget',
+  pypi: 'pypi',
+  rpm: 'rpm'
+};
+
 export const REPOS = {
   alpineLinux: 'alpineLinux',
   cratesIo: 'cratesIo',
-}
+  anaconda: 'anaconda',
+  chocolatey: 'chocolatey',
+  clojars: 'clojars',
+  cocoapods: 'cocoapods',
+  conan: 'conan',
+  cran: 'cran',
+  debianpackages: 'debianpackages',
+  debiantracker: 'debiantracker',
+  godev: 'godev',
+  mavenrepo1: 'mavenrepo1',
+  mavenapache: 'mavenapache',
+  mavencentral: 'mavencentral',
+  mvnrepository: 'mvnrepository',
+  npmJS: 'npmjs',
+  nuget: 'nuget',
+  packagist: 'packagist',
+  pypi: 'pypi',
+  rubygems: 'rubygems',
+  nexusepo: 'nexusrepo',
+  artifactory: 'artifactory'
+};
```

Comment:
These are all implemented in`RewriteInReactFunZone` now, merge the newest and update, sorry, I didn't see this PR before I went and did it (add @bhamail and I as reviewers if you can when you open the PR)

Reply:
Got it, will make sure to add you both as reviewers in future. I will merge this and fix in my PR